test/types/recurly-error.ts

```diff
@@ -0,0 +1,10 @@
+import { RecurlyError } from '../../types';
```

Comment:
This test shows what I'm trying to achieve. With this change, we can pull `RecurlyError` and other types out the definition and use them as needed.